.agents/skills/write-acceptance-test/SKILL.md

```
@@ -0,0 +1,235 @@
+---
+name: write-acceptance-test
+description: Write or complete a Cucumber/Gherkin acceptance test scenario for any feature in the Goat It API, following the conventions in tests/acceptance/README.md (fixtures, payloads, step definitions, DataTable schemas, locale helpers).
+disable-model-invocation: true
+---
+
+## What I do
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)